bigchaindb/common/transaction.py

```diff
@@ -508,20 +503,20 @@ def validate_asset(self, amount=None):
                                   ' greater than one')
 
 
-class AssetLink(Asset):
+class AssetLink(object):
```

Comment:
Is this guy still needed? What about just having a dict rather than a whole object to store an ID?